### DISCUSSION TITLE
Example course with executable code as quiz question?

### DISCUSSION THREAD

**Discussion by rosemm** [LiaScript/LiaScript#146] [Q&A] [answered]
I heard in the excellent symposium today about a course that uses executable code blocks as quiz questions. If someone knows the link for a course doing that, could you please share?

**Comment by rosemm** [comment, marked as answer]
Andre responded already in the symposium chat :) Thank you!
Putting this link here for anyone else interested: https://github.com/LiaScript/LiaScript-Exporter/issues/37

**Comment by andre-dietrich** [comment]
At this stage it is a simple "hack" that can be used to merge a generic quiz with an executable code snippet ... in the future, we will try to make the [[!]] generic-quiz attachable to other LiaScript/Markdown blocks ;-)
